pkg/client/client.go

```diff
@@ -258,6 +258,10 @@ func (c *Client) ApiServersListWatchForResource(ctx context.Context, resource st
 	}
 }
 
+func (c *Client) ConsoleListWatch() *cache.ListWatch {
+	return cache.NewListWatchFromClient(c.kclient.CoreV1().RESTClient(), "consoles", "", fields.Everything())
```

Comment:
you need `c.oscclient.ConfigV1().Consoles()` rather than `c.kclient.CoreV1().RESTClient()`.